jsx/components/Home.jsx

```diff
@@ -1,85 +1,52 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { connect } from 'react-redux';
 import Overview from '../components/Overview.jsx';
 import CouncilorList from '../components/CouncilorList.jsx';
 import OrderList from '../components/OrderList.jsx';
+import { fetchCouncilors } from '../actions/councilor.jsx';
+import store from '../main.jsx';
+
+class Home extends React.Component {
 
-const COUNCILORS = [
-  {
-    "name":"Justin Costa",
-    "slug":"justin-costa",
-    "role":"District 4",
-    "citypage":"http://www.portlandmaine.gov"
-  },
-  {
-    "name":"Ethan K. Strimling",
-    "slug":"ethan-k-strimling",
-    "role":"Mayor",
-    "citypage":"http://www.portlandmaine.gov"
-  },
-  {
-    "name":"Jill C. Duson",
-    "slug":"jill-c-duson",
-    "role":"At Large",
-    "citypage":"http://www.portlandmaine.gov"
-  },{
-    "name":"Nicholas M. Mavodones, Jr.",
-    "slug":"nicholas-m-mavodones-jr",
-    "role":"At Large",
-    "citypage":"http://www.portlandmaine.gov"
-  },
-  {
-    "name":"Brian E. Batson",
-    "slug":"brian-e-batson",
-    "role":"District 3",
-    "citypage":"http://www.portlandmaine.gov"
-  },{
-    "name":"Spencer Thibodeau",
-    "slug":"spencer-thibodeau",
-    "role":"District 2",
-    "citypage":"http://www.portlandmaine.gov"
-  },{
-    "name":"Belinda S. Ray",
-    "slug":"belinda-s-ray",
-    "role":"District 1",
-    "citypage":"http://www.portlandmaine.gov"
-  },{
-    "name":"David Brenerman",
-    "slug":"david-brenerman",
-    "role":"District 5",
-    "citypage":"http://www.portlandmaine.gov"
-  },{
-    "name":"Pious Ali",
-    "slug":"pious-ali",
-    "role":"At Large",
-    "citypage":"http://www.portlandmaine.gov"
+  componentWillMount() {
+    this.props.dispatch(fetchCouncilors())
```

Comment:
We shouldn't fetch councilors unless we don't have them in the store or we have reason to think they've changed.

Reply:
I moved the initial data fetching to `main.jsx`—let me know if this looks better!